addons/scene_palette/palette.gd

```diff
@@ -90,13 +89,24 @@ func _reload_palette():
 
 ## Recursively create subpalettes for the specified directory
 func _populate_scenes(sub_palette:PalettePluginSubPalette, dir_path:String):
+	var save_data:PalettePluginSaveData = _get_save_data()
+	var minimized_dirs: Dictionary
+
+	if _current_dir_in_favorites():
+		var favorite = save_data.favorites[_current_dir]
+		minimized_dirs = favorite.get("minimized_dirs", {})
+	else:
+		minimized_dirs = {}
+
 	var dir = DirAccess.open(dir_path)
 	if dir:
 		for dir_name in dir.get_directories():
 			var new_sub_palette:PalettePluginSubPalette = subpalette_scene.instantiate()
 			var path = dir_path + '/' + dir_name
 			sub_palette.add_subpalette(new_sub_palette)
 			new_sub_palette.directory = path
+			new_sub_palette.minimized = minimized_dirs.has(dir_name)
+			new_sub_palette.minimize_changed.connect(_on_sub_palette_minimize_changed.bind(dir_name))
```

Comment:
```suggestion
			new_sub_palette.minimized = minimized_dirs.has(path)
			new_sub_palette.minimize_changed.connect(_on_sub_palette_minimize_changed.bind(path))
```
Need to use the full path or it will mix up subfolders in different directories with the same name